packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -339,16 +338,17 @@ def expected_service_running(
         service_running = ServiceRunning(iframe_locator=None)
 
         try:
+
             with websocket.expect_event("framereceived", waiter, timeout=timeout):
                 if press_start_button:
                     _trigger_service_start(page, node_id)
 
                 yield service_running
 
         except PlaywrightTimeoutError:
-            if waiter.get_last_call_return() is False:
+            if waiter.got_expected_node_progress_types():
```

Comment:
Q: what was wrong with the previous implementation?

Reply:
wrong condition